tests/clients/test_ngsi_v2_iota.py

```diff
@@ -177,15 +178,16 @@ def test_deletions(self):
                         entity_name=entity_id,
                         entity_type='Thing2',
                         protocol='IoTA-JSON',
-                        transport='MQTT',
+                        transport='HTTP',
```

Comment:
doesn't this make other tests fail? you are not using any HTTP here, right?

Reply:
This object is only used in this specific test, it is locally created. I changed it to test out if it was the source of an issue that I was having, but that was not the case.
For this test it has no influence which transport was used, I just simply did not change it back, as it is not important.